docs/characterization/xeb_theory.ipynb

```diff
@@ -0,0 +1,454 @@
+{
```

Comment:
<p>nit: expectation <em>value</em> of $O_U$</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3669/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>done</p>